packages/sdk/client/src/echo/space-proxy.ts

```diff
@@ -435,6 +451,12 @@ export class SpaceProxy implements Space {
     });
   }
 
+  private async _migrate() {
```

Comment:
why this vs space.internal.migrate?

Reply:
This is exposed as `space.internal.migrate`